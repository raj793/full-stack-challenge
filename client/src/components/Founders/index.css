.founders-container {
  display: flex;
  flex-direction: row;
  height: fit-content;
  width: fit-content;
  min-height: 100px;
  min-width: 70vw;
  border: 1px #FFFFFF solid;
  align-self: center;
  justify-content: space-evenly;
  align-items: center;
  color: #FFFFFF;
  flex-wrap: wrap;
}

.f-button {
  display: inline-flex;
  color: white;
  opacity: 0.5;
  font-family: Arial, Helvetica, sans-serif;
  font-size: 20px;
  font-weight: 900;
  justify-content: center;
  align-items: center;
}

.f-button:hover {
  opacity: 1;
  cursor: pointer;
}

.founder {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 20px;
}