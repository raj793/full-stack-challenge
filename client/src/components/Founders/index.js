import React from 'react';
import './index.css';

const Founders = ({founders, onClick}) => {

  return (
    <div className="founders-container">
      {founders && founders.map((founder) => {
        return (<div className="founder" key={founder.id}>{founder.name} : {founder.title}</div>);
      })}
      <div className="f-button" onClick={onClick}>Add Founder</div>
    </div>
  )
}

export default Founders;