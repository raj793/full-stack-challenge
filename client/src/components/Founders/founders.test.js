import React from 'react';
import renderer from 'react-test-renderer';
import Founders from './index';

test('Test Founders snapshot', () => {
  const founderData = [{
    name: "Jessica",
    title: "CEO",
    id: 1
  }];
  const component = renderer.create(
    <Founders founders={founderData} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});