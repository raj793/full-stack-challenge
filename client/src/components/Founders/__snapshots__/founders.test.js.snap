// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Founders snapshot 1`] = `
<div
  className="founders-container"
>
  <div
    className="founder"
  >
    Jessica
     : 
    CEO
  </div>
  <div
    className="f-button"
  >
    Add Founder
  </div>
</div>
`;
